projects/noloan/app/src/main/java/hasadna/noloan/SpamRecyclerAdapter.java

```diff
@@ -77,7 +77,8 @@ public void bind(SmsMessage sms) {
       from.setText(sms.getSender());
       content.setText(sms.getBody());
       receivedAt.setText(sms.getReceivedAt());
-      // Click on a spam message, from there (with message's details) move to the lawsuitPdfActivity
+      // Click on a suggestions message, from there (with message's details) move to the
```

Comment:
Would it be more accurate to say:
`Click on a message`
because it can be `spam` or `suggestion` (or maybe also none of them) depending on the message status?

Reply:
Yes. I think the auto-refactoring I did with renaming changed the comments as well. I'll have a look at them again